__tests__/data/appsflyer_source_output.json

```diff
@@ -0,0 +1,141 @@
+      "custom_data": null,
+      "contributor_2_touch_type": null,
+      "gp_install_begin": null,
+      "city": "Khu Pho Binh Hoa",
+      "amazon_aid": null,
+      "gp_referrer": null,
+      "af_cost_model": null,
+      "af_c_id": null,
+      "attributed_touch_time_selected_timezone": null,
+      "selected_currency": "USD",
+      "app_name": "AppsFlyer",
+      "install_time_selected_timezone": "2019-01-20 04:51:16.000+0000",
+      "postal_code": "823941",
+      "wifi": true,
+      "install_time": "2019-01-20 04:51:16.000",
+      "operator": null,
+      "attributed_touch_type": null,
+      "af_attribution_lookback": null,
+      "campaign_type": null,
+      "keyword_match_type": null,
+      "af_adset_id": null,
+      "device_download_time_selected_timezone": "2019-01-20 04:51:16.000+0000",
+      "contributor_2_media_source": null,
+      "conversion_type": null,
+      "contributor_2_match_type": null,
+      "api_version": "2.0",
+      "attributed_touch_time": null,
+      "revenue_in_selected_currency": null,
+      "is_retargeting": false,
+      "country_code": "VN",
+      "gp_click_time": null,
+      "contributor_1_af_prt": null,
+      "match_type": null,
+      "appsflyer_id": "1547985076649-5309999",
+      "dma": "None",
+      "http_referrer": null,
+      "af_sub5": null,
+      "af_prt": null,
+      "event_revenue_currency": "USD",
+      "store_reinstall": null,
+      "install_app_store": null,
+      "media_source": "organic",
+      "deeplink_url": null,
+      "campaign": null,
+      "af_keywords": null,
+      "region": "AS",
+      "cost_in_selected_currency": null,
+      "event_value": "{}",
+      "ip": "1.1.1.1",
+      "oaid": null,
+      "event_time": "2020-01-15 14:57:24.898",
+      "is_receipt_validated": null,
+      "contributor_1_campaign": null,
+      "af_sub4": null,
+      "imei": null,
+      "contributor_3_campaign": null,
+      "event_revenue_usd": null,
+      "af_sub2": null,
+      "original_url": null,
+      "contributor_2_campaign": null,
+      "android_id": null,
+      "contributor_3_media_source": null,
+      "af_adset": null,
+      "af_ad": null,
+      "state": "57",
+      "network_account_id": null,
+      "device_type": "iPhoneXR",
+      "idfa": "A7071198-3848-40A5-B3D0-94578D9BZZZZ",
+      "retargeting_conversion_type": null,
+      "af_channel": null,
+      "af_cost_currency": null,
+      "contributor_1_media_source": null,
+      "keyword_id": null,
+      "device_download_time": "2019-01-20 04:51:16.000",
+      "contributor_1_touch_type": null,
+      "af_reengagement_window": null,
+      "af_siteid": null,
+      "language": "en-US",
+      "app_id": "id1217828636",
+      "contributor_1_touch_time": null,
+      "event_revenue": null,
+      "af_ad_type": null,
+      "carrier": null,
+      "event_name": "My Apps",
+      "af_sub_siteid": null,
+      "advertising_id": null,
+      "os_version": "12.3.1",
+      "platform": "ios",
```

Comment:
Expected: `message.platform` and in `message.context.os.name`

Also,

```
if (platform === "ios") {
  context.device.advertisingId = <value of `idfa`>;
} else {
  context.device.advertisingId = <value of `android_id`>;
}

if (context.device.advertisingId) {
  context.device.adTrackingEnabled = true;
}
```

Reply:
done